web/settings/context.go

```diff
@@ -47,6 +54,40 @@ func onboarded(c echo.Context) error {
 		}
 	}
 	redirect := i.OnboardedRedirection().String()
+
+	// Redirect to permissions screen if we are in a mobile onboarding
+	if i.OnboardingSecret != "" {
+		// Generate the deeplink
+		deeplink := fmt.Sprintf("cozy%s://%s", i.OnboardingApp, i.Domain)
+
+		// Generate the OAuth client
+		OAuthClient := oauth.Client{
+			RedirectURIs: []string{deeplink},
+			ClientName:   i.OnboardingApp,
+			SoftwareID:   "github.com/cozy/cozy-stack",
+		}
+		OAuthClient.Create(i)
+		i.OnboardingClientID = OAuthClient.ClientID
```

Comment:
In our sequence diagram, it's the cloudery that calls the stack to create the OAuth client